public/v4/apps/silex-platform.yml

```diff
@@ -0,0 +1,69 @@
+            STORAGE_CONNECTORS: $$cap_storage_connector
+            HOSTING_CONNECTORS: $$cap_hosting_connector
+            GITLAB_DOMAIN: https://gitlab.com
+            GITLAB_CLIENT_ID: $$cap_gitlab_client_id
+            GITLAB_CLIENT_SECRET: $$cap_gitlab_client_secret
+            GITLAB2_DOMAIN: $$cap_gitlab2_domain
+            GITLAB2_CLIENT_ID: $$cap_gitlab2_client_id
+            GITLAB2_CLIENT_SECRET: $$cap_gitlab2_client_secret
+            GITLAB_SCOPE: 'api read_user'
+            GITLAB_TIMEOUT: 5000
+        volumes:
+            # - '$$cap_appname-data:/srv/data'
+            # - '$$cap_appname-cfg:/srv/cfg'
+        caproverExtra:
+            containerHttpPort: '6805'
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
+          label: silex-platform Version
+          defaultValue: 'v1.0.162'
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/silexlabs/silex-platform/tags
+          validRegex: "/^([^\\s^\\/])+$/"
+        - id: '$$cap_tz'
+          label: Time Zone
+          defaultValue: Europe/Paris
+          description: Get yours from https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+          validRegex: '/.{1,}/'
+        - id: '$$cap_storage_connector'
+          label: STORAGE_CONNECTOR
+          defaultValue: 'gitlab'
+          description: Storage connector you want to use should be one or more of following gitlab,ftp,gitlab2 (separated with comma but no space)
+          validRegex: '/^((gitlab|ftp|gitlab2)\,?)*$/'
+        - id: '$$cap_hosting_connector'
+          label: HOSTING_CONNECTOR
+          defaultValue: 'gitlab,download'
+          description: Hosting connector you want to use should be one or more of following gitlab,ftp,gitlab2,download (separated with comma but no space)
+          validRegex: '/^((gitlab|ftp|gitlab2|download)\,?)*$/'
+        - id: '$$cap_gitlab_client_id'
+          label: GITLAB_CLIENT_ID
+          description: Your client-id for your gitlab app
+        - id: '$$cap_gitlab_client_secret'
+          label: GITLAB_CLIENT_SECRET
+          description: Your client-secret for your gitlab app
+        - id: '$$cap_gitlab2_domain'
+          label: GITLAB2_DOMAIN
+          description: Domain for your other gitlab instance (e.g. https://framagit.org)
+        - id: '$$cap_gitlab2_client_id'
+          label: GITLAB2_CLIENT_ID
+          description: Your client-id for your other gitlab instance app
+        - id: '$$cap_gitlab2_client_secret'
+          label: GITLAB2_CLIENT_SECRET
+          description: Your client-secret for your other gitlab instance app
+    instructions:
+        start: |-
+            Silex, is a free and open source website builder in the cloud. Create websites directly in the browser without writing code. And it is suitable for professional designers to produce great websites without constraints. Silex is also known as the HTML5 editor.
```

Comment:
Maybe add a link to the env vars docs ?

https://docs.silex.me/en/dev/options

Reply:
And why not explain that they can selfhost their gitlab instance too so that they have a full selfhosted web builder

And also let the users know they probably want to use a CMS like wordpress or others, they can self host it too
https://docs.silex.me/en/user/cms

Reply:
done